### PR TITLE
create user interfaces

### PR DESCRIPTION
**Pull request by chiztechnology**
hello.
kindly find the work we submitted in this pull request regarding the second milestone of the project.
we created a great design similar to the calculator indicated in the requirements.

Kindly check and let us know if anything requires improvement.

Best regards.

### CODE REVIEW

**Review by ha-manel — COMMENTED** [review]
# STATUS: INVALID 🔴 
Hi @chiztechnology 
I'm sure you put a lot of effort into this project but this PR is invalid because your README file is messy. Some sections are redundant and some of them don't hold information about the project it's just placeholder text. 

Kindly fix the issue and submit the PR for review again 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chiztechnology** [comment]
hello dear @ha-manel 
thank you for your great descriptive review.
I just committed the changes in my readme file.
I submit it for another review.
Thanks, a lot

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      <div className='component-app'>
+        {/* display zone */}
+        <div style={{ width: 300 }}>
+          <h3 style={{ paddingLeft: 15 }}>000</h3>
+        </div>
+        {/* buttons and signs */}
+        <ButtonPanel />
+      </div>
+    );
+  }
```
Comment (BertrandConxy) [inline src/components/Calculator.js]: - [ ] Good job so far 👏 In addition to what you have done, I would suggest that you improve your design in order to match the design of the calculator given. Please take a look at the comparison below 👇 
**Given design**
image

**Your design**
image

As you can see there are places that you still have to work on so kindly improve that too 😃 👍

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @chiztechnology, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used
- good PR


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chiztechnology** [comment]
thank you for your great review, dear @BertrandConxy  😊 
I really appreciate it.
I just updated the code to make my design match the required model.

Screenshot 2023-01-21 at 16 35 55

**Comment by BertrandConxy** [comment]
That's great
Good luck

**Review by jerryowusu — APPROVED** [review]
Hi @chiztechnology , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

CookedDelectableDolphin-size_restricted


## Highlights
Great effort in making the changes :heavy_check_mark: 
All linters are passing :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**